Formula/acmetool.rb

```diff
@@ -187,6 +43,21 @@ def post_install
     (var/"run/acmetool").mkpath
   end
 
+  def caveats
+    <<~EOS
+      As of June 2020, subscribers to the Let's Encrypt service are no longer allowed
+      to validate new domains through the ACMEv1 protocol, and renewal of existing
+      certificates will occasionally be disabled starting early 2021, before the final
+      EoL of ACMEv1 by June 2021.
+
+      As as result, new users should only use this version, which only supports ACME
+      v1, to import their existing \x1B[3m\x1B[1mcertbot\x1B[0m configurations. Current users should also
+      consider upgrading to ACMEv2 by next year, though this has to be done manually
+      for the time being. Please follow \x1B[4mhttps://github.com/hlandau/acmetool/issues/322\x1B[0m
+      for upgrade instructions and discussion on the future of acmetool.
```

Comment:
Can we make this a bit shorter? I suggest,

> This formula should only be used to migrate existing ACMEv1 configurations to ACMEv2, as this will be disabled in June 2021. See https://github.com/hlandau/acmetool/issues/322

We can also add a `deprecate! :date => "2021-07-01"` so users will get a warning when installing this.

Reply:
Oh wow, there's a `deprecate!` command? That'll help right now. Part of the idea with this huge paragraph is to try attract enough attention when the user is upgrading or batch installing, so it stands out a bit at the end (while sounding like warning messages found elsewhere in Homebrew). It's even better to tell users rightaway while installing.
> `This version is deprecated, unable to work for new subscribers and is kept as`
> `newer versions would break on running systems using ACMEv1 (which will stop`
> `working eventually by June 2021). A manual migration is required to use ACMEv2`
> `for now. Follow`[`https://github.com/hlandau/acmetool/issues/322`](https://github.com/hlandau/acmetool/issues/322)`for details.`